docs/book/v2/installation.md

```diff
@@ -0,0 +1,5 @@
+# This is only a placeholder
```

Comment:
This formulation is a bit confusing for a user: perhaps "This page has moved"?

Reply:
No user will ever see this in the rendered documentation.